packages/integrations/markdoc/src/html/transform/html-token-transform.ts

```diff
@@ -1,5 +1,6 @@
 import type { Tokenizer } from '@markdoc/markdoc';
 import { Parser } from 'htmlparser2';
+// @ts-expect-error This type isn't exported
 import type * as Token from 'markdown-it/lib/token';
```

Comment:
Couldn't find this type in the public API at all, it seems like the only reason we were able to get it previously was from a raw file path.